pkg/controllers/scheduling/scheduling_controller.go

```diff
@@ -147,28 +148,48 @@ func NewSchedulingController(
 		}, placementDecisionInformer.Informer()).
 		WithBareInformers(clusterInformer.Informer(), clusterSetInformer.Informer(), clusterSetBindingInformer.Informer()).
 		WithSync(c.sync).
+		ResyncEvery(5*time.Minute).
 		ToController(schedulingControllerName, recorder)
 }
 
 func (c *schedulingController) sync(ctx context.Context, syncCtx factory.SyncContext) error {
 	queueKey := syncCtx.QueueKey()
-	namespace, name, err := cache.SplitMetaNamespaceKey(queueKey)
-	if err != nil {
-		// ignore placement whose key is not in format: namespace/name
-		utilruntime.HandleError(err)
-		return nil
-	}
-
 	klog.V(4).Infof("Reconciling placement %q", queueKey)
-	placement, err := c.placementLister.Placements(namespace).Get(name)
-	if errors.IsNotFound(err) {
-		// no work if placement is deleted
+
+	if queueKey == "key" {
+		placements, err := c.placementLister.List(labels.Everything())
+		if err != nil {
+			return err
+		}
+
+		klog.V(4).Infof("Resyncing %d placements", len(placements))
+		for _, placement := range placements {
```

Comment:
this could be pretty slow. Maybe you should just requeue them

Reply:
@qiujian16  The above code wants to reschedule the placements in case AddOnPlacementScore changes. Not sure do you mean using requeue to handle that? or requeue is just for placements that don't have a decision?

Reply:
i mean add those placement to workqueue instead of process all of them in one reconcile loop

Reply:
@qiujian16 thanks for the comment, fixed.